src/store/actions/listActions.js

```diff
@@ -7,9 +24,30 @@ export const addTask = task => ({
     }
 });
 
+
+export const startAddTask = task => (
+    (dispach, getState) => {
+        const { tasks } = getState().list
+        if(localStorage) {
+            localStorage.setItem('tasks', JSON.stringify([...tasks, task]))
+        }
+        dispach(addTask(task));
+    }
+)
+
 export const removeTask = id => ({
     type: REMOVE_TASK,
     payload: {
         id,
     }
 });
+
+export const startRemoveTask = id => (
+    (dispach, getState) => {
+        const tasks = getState().list.tasks.filter((task, taskId) => taskId !== id );
+        if(localStorage) {
```

Comment:
this piece of code seems to be redundant. That's why I requested special helper -> it should be created a file with helpers or utils, where all localStorage relations are managed. Then, only import here this function and use.